pkg/render/logstorage.go

```diff
@@ -586,6 +614,113 @@ func (es elasticsearchComponent) podTemplate() corev1.PodTemplateSpec {
 		volumes = es.dexCfg.RequiredVolumes()
 	}
 
+	var autoMountToken bool
+	if es.installation.CertificateManagement != nil {
+
+		// If certificate management is used, we need to override a mounting options for this init container.
+		initFSName := "elastic-internal-init-filesystem"
+		initFSContainer := corev1.Container{
+			Name:  initFSName,
+			Image: es.esImage,
+			SecurityContext: &corev1.SecurityContext{
+				Privileged: ptr.BoolToPtr(true),
```

Comment:
Can you double check with the security team that we can do this?

Reply:
They are aware. This is not new. The init container has always existed and has always been privileged. For this reason we maintain a table/spreadsheet somewhere in the docs to show our progress on privileged containers where we have an entry for Elasticsearch.

Reply:
Actually, it works without it being privileged, so I will address this, thanks.